AlwaysOnKotlin/compose/src/main/java/com/example/android/wearable/wear/alwayson/AlwaysOnScreen.kt

```diff
@@ -83,19 +84,21 @@ fun AlwaysOnScreen(
             .verticalScroll(rememberScrollState())
             .fillMaxRectangle()
             .padding(with(LocalDensity.current) { BURN_IN_OFFSET_PX.toDp() })
-            .burnInTranslation(ambientState, ambientPing)
+            .burnInTranslation(ambientState, ambientUpdateTimestamp)
     ) {
         Text(
+            modifier = Modifier.testTag("time"),
             text = dateFormat.format(currentTime),
             style = MaterialTheme.typography.title1,
-            modifier = Modifier.testTag("time")
+            textAlign = TextAlign.Center
         )
         Text(
+            modifier = Modifier.testTag("timestamp"),
```

Comment:
Still missing textAlign = TextAlign.Center?

Maybe it isn't needed for body2?

Reply:
Oops, I just forgot this one!